code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -14,9 +14,9 @@
 	mutantlungs = /obj/item/organ/lungs/slime
 	damage_overlay_type = ""
 	liked_food = MEAT
-	coldmod = 6
-	heatmod = 0.5
-	burnmod = 0.5 // = 1/2x generic burn damage
+	coldmod = 3
+	heatmod = 0.8
+	burnmod = 0.8 // = 1/2x generic burn damage
```

Comment:
You can probably leave heatmod as is, its not that relevant
Also, please update the comment

Reply:
fixed